packages/ui-kit/src/Tooltip/TooltipContent/TooltipContent.tsx

```diff
@@ -1,18 +1,26 @@
 import clsx from 'clsx';
 import { Close16 } from '@konturio/default-icons';
+import { forwardRef } from 'react';
 import s from './TooltipContent.module.css';
+import type { MouseClickEvent } from '../Tooltip/Tooltip';
 import type { PropsWithChildren } from 'react';
 
-export function TooltipContent({
-  children,
-  onClose,
-  className,
-}: PropsWithChildren<{
-  onClose?: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
-  className?: string;
-}>) {
+export const TooltipContent = forwardRef<
```

Comment:
because of prop type duplication looks so verbose 
tried to improve a bit: 

```
export const TooltipContent = forwardRef<HTMLDivElement>(
  (
    {
      children,
      onClose,
      className,
    }: PropsWithChildren<{
      onClose?: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
      className?: string;
    }>,
    ref,
  ) => {
    return (
      <div ref={ref} className={clsx(s.tooltipContent, className)}>
        {children}
        {onClose && (
          <div className={s.closeIcon} onClick={onClose}>
            <Close16 />
          </div>
        )}
      </div>
    );
  },
);

TooltipContent.displayName = 'TooltipContent';

```